packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2297,6 +2298,19 @@ export default class Meeting extends StatelessWebexPlugin {
         mediaDirection.sendShare
       )
     ) {
+      if (mediaDirection &&
+        (
+          mediaDirection.sendAudio &&
+          mediaDirection.sendVideo &&
+          mediaDirection.sendShare
+        ) &&
+        bowser.safari
+      ) {
+        mediaDirection.sendShare = false;
```

Comment:
You should probably send another log stating that you're overriding the media direction for share